chore(i18n): Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Thunderbird for Android / K-9 Mail/ios-localization](https://hosted.weblate.org/projects/tb-android/ios-localization/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/tb-android/ios-localization/horizontal-auto.svg)

standaell1234-maker reviewed (commented):
Standaell1234@gmail.com

weblate committed:
chore(i18n): Update translations (Weblate)

Co-authored-by: 109247019824 <109247019824@users.noreply.hosted.weblate.org>
Co-authored-by: A. Wang <arielx@users.noreply.hosted.weblate.org>
Co-authored-by: AO Localisation Lab <ao_localization_lab@users.noreply.hosted.weblate.org>
Co-authored-by: AO yahoe.001 <ao_yahoe.001@users.noreply.hosted.weblate.org>
Co-authored-by: Adrien <adrien-lsh@42l.fr>
Co-authored-by: Aindriú Mac Giolla Eoin <aindriu80@gmail.com>
Co-authored-by: Alexander Gabilondo <alexgabi@openmailbox.org>
Co-authored-by: Andi Chandler <andi@gowling.com>
Co-authored-by: Andreas Pettersson <amvpet@gmail.com>
Co-authored-by: Ardev Prisec <prisecardev@gmail.com>
Co-authored-by: Arif Budiman <arifpedia@gmail.com>
Co-authored-by: Balázs Meskó <meskobalazs@mailbox.org>
Co-authored-by: Besnik Bleta <besnik@programeshqip.org>
Co-authored-by: Bogomil Shopov <bogomil.fedora@gmail.com>
Co-authored-by: ButterflyOfFire <boffire@users.noreply.hosted.weblate.org>
Co-authored-by: Bård Sigurd Møller <git@bsmoller.no>
Co-authored-by: Carlos Jorge Santos <cjsantos@sapo.pt>
Co-authored-by: Danial Behzadi <dani.behzi@ubuntu.com>
Co-authored-by: Ettore Atalan <atalanttore@googlemail.com>
Co-authored-by: Fjuro <fjuro@alius.cz>
Co-authored-by: Fjuro <fjuro@users.noreply.hosted.weblate.org>
Co-authored-by: Francisco Serrador <fserrador@gmail.com>
Co-authored-by: Fulup Jakez <fulup.jakez@gmail.com>
Co-authored-by: Guillermo Badia <guillermo.badia@gmail.com>
Co-authored-by: HUMET <ressaguer@proton.me>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Huldar <huldarh@gmail.com>
Co-authored-by: Igor Rückert <igorruckert@yahoo.com.br>
Co-authored-by: Ihor Hordiichuk <igor_ck@outlook.com>
Co-authored-by: Jim Spentzos <jimspentzos2000@gmail.com>
Co-authored-by: Jimmy Young <lnddyl@outlook.com>
Co-authored-by: Jiri Grönroos <jiri.gronroos@iki.fi>
Co-authored-by: João Pedro Nunes <nunes@disroot.org>
Co-authored-by: Kazuki Nakayama <phys314159265358979chem@gmail.com>
Co-authored-by: Kazushi Hayama <hosted.weblate.3yoe@1984.pmail.li>
Co-authored-by: Marcelo Poli <enzomatrix@gmail.com>
Co-authored-by: Marek Dědič <register@dedic.eu>
Co-authored-by: Matthaiks <kitynska@gmail.com>
Co-authored-by: Mauro Giulio Bartoccelli <bartokk@gmail.com>
Co-authored-by: Michael Bauer <teanga@igaidhlig.net>
Co-authored-by: Márk <senta.sft@gmail.com>
Co-authored-by: Nico <n2778370@gmail.com>
Co-authored-by: Oğuz Ersen <oguz@ersen.moe>
Co-authored-by: Patrick Santa Maria <Patrick.Santa-Maria@laposte.net>
Co-authored-by: Peter Klofutar <peter.klofutar@gmail.com>
Co-authored-by: Peter Nemšák <peter.nemsak@gmail.com>
Co-authored-by: Priit Jõerüüt <jrthwlate@users.noreply.hosted.weblate.org>
Co-authored-by: Rafael Tonholo <rtonholo@thunderbird.net>
Co-authored-by: Real Ceday <realceday@gmail.com>
Co-authored-by: Remus-Gabriel Chelu <remusgabriel.chelu@disroot.org>
Co-authored-by: Reza Zarchi <reza.zarchi@gmail.com>
Co-authored-by: Rhoslyn Prys <rprys@posteo.net>
Co-authored-by: Romhányi-Kakucska Viktor <viktor@romhanyi.dev>
Co-authored-by: Selim Şumlu <selim@users.noreply.hosted.weblate.org>
Co-authored-by: Shah Zaman Pathan <pafghan043@gmail.com>
Co-authored-by: Simon T <simonpmt@gmail.com>
Co-authored-by: Sketch6580 <sketch6580@users.noreply.hosted.weblate.org>
Co-authored-by: Sumon Kayal <sumankayalsuman4@gmail.com>
Co-authored-by: Sveinn í Felli <sv1@fellsnet.is>
Co-authored-by: Swyter <swyterzone@gmail.com>
Co-authored-by: TV Box <realceday.tvbox@gmail.com>
Co-authored-by: Tao Mon Lae <tamonlae@gmail.com>
Co-authored-by: Telaneo <post@telaneo.net>
Co-authored-by: Tom Golden <tbjgolden@gmail.com>
Co-authored-by: UDP <udp@users.noreply.hosted.weblate.org>
Co-authored-by: V DR <vdeliceroland@gmail.com>
Co-authored-by: VfBFan <drop0815@posteo.de>
Co-authored-by: VfBFan <vfbfan@users.noreply.hosted.weblate.org>
Co-authored-by: Vikas Sharma <vikassharma333666@gmail.com>
Co-authored-by: Wim Benes <fryskefirefox@gmail.com>
Co-authored-by: Wolf-Martell Montwé <wolf@thunderbird.net>
Co-authored-by: Yury Vashchylau | Tech @ yuryv.info <vashchylau@users.noreply.hosted.weblate.org>
Co-authored-by: Yury Vashchylau | Tech @ yuryv.info <yury@digitalby.me>
Co-authored-by: duub qnnp <qnnp@duub.org>
Co-authored-by: ginger-co <ginger_co@outlook.com>
Co-authored-by: gymka <gymka@archlinux.lt>
Co-authored-by: jules Loup <zrlkpgvp@anonaddy.me>
Co-authored-by: justcontributor <kty5663@gmail.com>
Co-authored-by: khangreat <khangreat@users.noreply.hosted.weblate.org>
Co-authored-by: khkarthik 4 <khkarthik4@gmail.com>
Co-authored-by: lkolis <loliskolis45@gmail.com>
Co-authored-by: nautilusx <translate@disroot.org>
Co-authored-by: rcasl <rcasl@outlook.com>
Co-authored-by: solokot <solokot@gmail.com>
Co-authored-by: Максим Горпиніч <gorpinicmaksim0@gmail.com>
Co-authored-by: 大王叫我来巡山 <hamburger2048@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/be/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/be_Latn/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/bg/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/bn/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/br/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ca/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/co/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/cs/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/cy/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/da/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/de/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/el/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/en_GB/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/es/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/et/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/eu/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/fa/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/fi/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/fr/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/fy/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ga/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/gd/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/gl/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/he/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/hi/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/hu/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/id/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/is/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/it/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ja/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/kab/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ko/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/lt/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/mnw/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/nl/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/nn/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/pl/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/pt_PT/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ro/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ru/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/sk/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/sl/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/sq/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/sv/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/tr/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/uk/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/zh_Hant/
Translation: Thunderbird for Android / K-9 Mail/ios-localization

weblate committed:
chore(i18n): Update translations (Weblate)

Co-authored-by: 109247019824 <109247019824@users.noreply.hosted.weblate.org>
Co-authored-by: A. Wang <arielx@users.noreply.hosted.weblate.org>
Co-authored-by: AO Localisation Lab <ao_localization_lab@users.noreply.hosted.weblate.org>
Co-authored-by: AO yahoe.001 <ao_yahoe.001@users.noreply.hosted.weblate.org>
Co-authored-by: Adrien <adrien-lsh@42l.fr>
Co-authored-by: Aindriú Mac Giolla Eoin <aindriu80@gmail.com>
Co-authored-by: Alexander Gabilondo <alexgabi@openmailbox.org>
Co-authored-by: Andi Chandler <andi@gowling.com>
Co-authored-by: Andreas Pettersson <amvpet@gmail.com>
Co-authored-by: Ardev Prisec <prisecardev@gmail.com>
Co-authored-by: Arif Budiman <arifpedia@gmail.com>
Co-authored-by: Balázs Meskó <meskobalazs@mailbox.org>
Co-authored-by: Besnik Bleta <besnik@programeshqip.org>
Co-authored-by: Bogomil Shopov <bogomil.fedora@gmail.com>
Co-authored-by: ButterflyOfFire <boffire@users.noreply.hosted.weblate.org>
Co-authored-by: Bård Sigurd Møller <git@bsmoller.no>
Co-authored-by: Carlos Jorge Santos <cjsantos@sapo.pt>
Co-authored-by: Danial Behzadi <dani.behzi@ubuntu.com>
Co-authored-by: Ettore Atalan <atalanttore@googlemail.com>
Co-authored-by: Fjuro <fjuro@alius.cz>
Co-authored-by: Fjuro <fjuro@users.noreply.hosted.weblate.org>
Co-authored-by: Francisco Serrador <fserrador@gmail.com>
Co-authored-by: Fulup Jakez <fulup.jakez@gmail.com>
Co-authored-by: Guillermo Badia <guillermo.badia@gmail.com>
Co-authored-by: HUMET <ressaguer@proton.me>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Huldar <huldarh@gmail.com>
Co-authored-by: Igor Rückert <igorruckert@yahoo.com.br>
Co-authored-by: Ihor Hordiichuk <igor_ck@outlook.com>
Co-authored-by: Jim Spentzos <jimspentzos2000@gmail.com>
Co-authored-by: Jimmy Young <lnddyl@outlook.com>
Co-authored-by: Jiri Grönroos <jiri.gronroos@iki.fi>
Co-authored-by: João Pedro Nunes <nunes@disroot.org>
Co-authored-by: Kazuki Nakayama <phys314159265358979chem@gmail.com>
Co-authored-by: Kazushi Hayama <hosted.weblate.3yoe@1984.pmail.li>
Co-authored-by: Marcelo Poli <enzomatrix@gmail.com>
Co-authored-by: Marek Dědič <register@dedic.eu>
Co-authored-by: Matthaiks <kitynska@gmail.com>
Co-authored-by: Mauro Giulio Bartoccelli <bartokk@gmail.com>
Co-authored-by: Michael Bauer <teanga@igaidhlig.net>
Co-authored-by: Márk <senta.sft@gmail.com>
Co-authored-by: Nico <n2778370@gmail.com>
Co-authored-by: Oğuz Ersen <oguz@ersen.moe>
Co-authored-by: Patrick Santa Maria <Patrick.Santa-Maria@laposte.net>
Co-authored-by: Peter Klofutar <peter.klofutar@gmail.com>
Co-authored-by: Peter Nemšák <peter.nemsak@gmail.com>
Co-authored-by: Priit Jõerüüt <jrthwlate@users.noreply.hosted.weblate.org>
Co-authored-by: Rafael Tonholo <rtonholo@thunderbird.net>
Co-authored-by: Real Ceday <realceday@gmail.com>
Co-authored-by: Remus-Gabriel Chelu <remusgabriel.chelu@disroot.org>
Co-authored-by: Reza Zarchi <reza.zarchi@gmail.com>
Co-authored-by: Rhoslyn Prys <rprys@posteo.net>
Co-authored-by: Romhányi-Kakucska Viktor <viktor@romhanyi.dev>
Co-authored-by: Selim Şumlu <selim@users.noreply.hosted.weblate.org>
Co-authored-by: Shah Zaman Pathan <pafghan043@gmail.com>
Co-authored-by: Simon T <simonpmt@gmail.com>
Co-authored-by: Sketch6580 <sketch6580@users.noreply.hosted.weblate.org>
Co-authored-by: Sumon Kayal <sumankayalsuman4@gmail.com>
Co-authored-by: Sveinn í Felli <sv1@fellsnet.is>
Co-authored-by: Swyter <swyterzone@gmail.com>
Co-authored-by: TV Box <realceday.tvbox@gmail.com>
Co-authored-by: Tao Mon Lae <tamonlae@gmail.com>
Co-authored-by: Telaneo <post@telaneo.net>
Co-authored-by: Tom Golden <tbjgolden@gmail.com>
Co-authored-by: UDP <udp@users.noreply.hosted.weblate.org>
Co-authored-by: V DR <vdeliceroland@gmail.com>
Co-authored-by: VfBFan <drop0815@posteo.de>
Co-authored-by: VfBFan <vfbfan@users.noreply.hosted.weblate.org>
Co-authored-by: Vikas Sharma <vikassharma333666@gmail.com>
Co-authored-by: Wim Benes <fryskefirefox@gmail.com>
Co-authored-by: Wolf-Martell Montwé <wolf@thunderbird.net>
Co-authored-by: Yury Vashchylau | Tech @ yuryv.info <vashchylau@users.noreply.hosted.weblate.org>
Co-authored-by: Yury Vashchylau | Tech @ yuryv.info <yury@digitalby.me>
Co-authored-by: duub qnnp <qnnp@duub.org>
Co-authored-by: ginger-co <ginger_co@outlook.com>
Co-authored-by: gymka <gymka@archlinux.lt>
Co-authored-by: jules Loup <zrlkpgvp@anonaddy.me>
Co-authored-by: justcontributor <kty5663@gmail.com>
Co-authored-by: khangreat <khangreat@users.noreply.hosted.weblate.org>
Co-authored-by: khkarthik 4 <khkarthik4@gmail.com>
Co-authored-by: lkolis <loliskolis45@gmail.com>
Co-authored-by: nautilusx <translate@disroot.org>
Co-authored-by: rcasl <rcasl@outlook.com>
Co-authored-by: solokot <solokot@gmail.com>
Co-authored-by: Максим Горпиніч <gorpinicmaksim0@gmail.com>
Co-authored-by: 大王叫我来巡山 <hamburger2048@users.noreply.hosted.weblate.org>
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/be/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/be_Latn/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/bg/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/bn/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/br/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ca/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/co/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/cs/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/cy/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/da/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/de/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/el/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/en_GB/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/es/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/et/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/eu/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/fa/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/fi/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/fr/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/fy/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ga/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/gd/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/gl/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/he/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/hi/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/hu/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/id/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/is/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/it/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ja/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/kab/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ko/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/lt/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/mnw/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/nl/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/nn/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/pl/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/pt_PT/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ro/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/ru/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/sk/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/sl/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/sq/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/sv/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/tr/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/uk/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/tb-android/ios-localization/zh_Hant/
Translation: Thunderbird for Android / K-9 Mail/ios-localization
